tests/rules/tcp-ack-keyword/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+    min-version: 7.0.0
+    pcap: false
+    
+args:
+    - --engine-analysis
+
+checks:
+- filter:
+    filename: rules.json
+    count: 1
+    match:
+      id: 1
+      lists.packet.matches[0].name: "tcp.ack"
```

Comment:
Would be good to have another line check here with the actual value that `ack` has in the line, (cf https://github.com/OISF/suricata-verify/tree/master/tests/rules/ipopts or other tests there, many of them check not only for the keyword itself, but also any value assigned to it.)

Reply:
Yes! I wanted to do that, but looking at the codes, I didn't see any parameter that would be used for it.

So I assumed it didn't exist.

Reply:
I don't think this test tests what you have done in your Suricata PR. That should have made this test fail. `tcp.ack` already exists in Suricata's matches which is why the rules as you have written work.
You should have checked for (alongwith what you already have) `lists.packet....ack.ack: 273`.

> I didn't see any parameter that would be used for it.

Because you just added it in your Suricata PR ;)